File: SampleSetup/Output/TypeScript/SampleProject/Protocol/Model/ApiAddress.ts

import { AbstractModelWithId } from './AbstractModelWithId';
import { ApiCountry } from './ApiCountry';
import { ApiCity } from './ApiCity';
export class ApiAddress extends AbstractModelWithId {
    CountryId: string;

    Country: ApiCountry;

    CityId: string;

    City: ApiCity;

    PostalCode: string;

    AddressLine1: string;

    AddressLine2: string;

    GeoLat: number | null;

    GeoLon: number | null;

    Description: string;
}

File: SampleSetup/Output/Swift/Model/ApiAddress.swift

    struct ApiAddress: Codable, AbstractModelWithId
 {
        var CountryId: String

        var Country: ApiCountry

        var CityId: String

        var City: ApiCity

        var PostalCode: String

        var AddressLine1: String

        var AddressLine2: String

        var GeoLat: Double?

        var GeoLon: Double?

        var Description: String
    }

